src/action.ts

```diff
@@ -84,14 +84,12 @@ export const runAction = async (space): Promise<void> => {
 
     const spaceKeys = await space.getApiKeys();
 
-    let exists = false;
-
-    spaceKeys.some(item => item.name === keyName)
+    let exists = spaceKeys.items.some(item => item.name === keyName)
```

Comment:
we're not changing exists anymore, so this can be const.

```suggestion
    const exists = spaceKeys.items.some(item => item.name === keyName)
```